CHANGELOG.next.toml

```
@@ -478,3 +478,9 @@ x-amzn-errortype: com.example.service#InvalidRequestException
 references = ["smithy-rs#1982"]
 meta = { "breaking" = true, "tada" = false, "bug" = false, "target" = "server" }
 author = "david-perez"
+
+[[smithy-rs]]
+message = "Make generated enum `values()` functions callable in const contexts."
+references = ["smithy-rs#2011"]
+meta = { "breaking" = false, "tada" = false, "bug" = false, "target" = "client" }
```

This change affects the server SDK too.

```suggestion
meta = { "breaking" = false, "tada" = false, "bug" = false, "target" = "all" }
```

---

Ah, apologies, took a guess at what `client`/`server` implied here. Pushed a fix.